### DISCUSSION TITLE
大佬，可以更新一下这个按颜色分类的模板嘛，已经失效不能用了

### DISCUSSION THREAD

**Discussion by yuzixieshouhuajiaqi** [windingwind/zotero-better-notes#718] [Q&A] [answered]
https://github.com/windingwind/zotero-better-notes/issues/85#issuecomment-1210063595
![Uploading image.png…]()

**Comment by windingwind** [comment, marked as answer]
Please see https://github.com/windingwind/zotero-better-notes/discussions/categories/note-templates

**Reply by yuzixieshouhuajiaqi** [reply]
感谢大佬，找到了
